### DISCUSSION TITLE
Cannot Link Excituable error

### DISCUSSION THREAD

**Discussion by sathwikcherott** [pystardust/ani-cli#924] [Q&A] [answered]
How can I find a solution for this?!  [Screenshot_2022-11-05-19-30-35-149_com.termux.jpg](https://user-images.githubusercontent.com/73953525/200123709-0c33f100-3c3d-4774-ab82-68e3e699b0ad.jpg)

**Comment by Rendevior** [comment]
Updating Packages can solve it sometimes, give it a shot.
```
apt update ; apt upgrade
```

**Comment by port19x** [comment, marked as answer]
You were likely missing relevant dependencies. 
Usually that gets caught by our dependency checks, weird
